src/components/kytos/napp/ActionMenuItem.vue

```diff
@@ -6,7 +6,60 @@
 
 <script>
 import Vue from 'vue'
-import httpVueLoader from "./httpVueLoader.js"
+import { loadModule } from "vue3-sfc-loader"
+ 
+const options = {
+
+  moduleCache: {
+    vue: Vue
+  },
+
+  pathResolve({ refPath, relPath }, options) {
+
+    if (relPath === '.')
+      return refPath;
+
+    if (relPath[0] !== '.' && relPath[0] !== '/')
+      return relPath;
+
+    return String(new URL(relPath, refPath === undefined ? window.location : refPath));
+  },
+
+  handleModule: async function (type, getContentData, path, options) {
+
+    switch (type) {
+      case '.css':
+        options.addStyle(await getContentData(false));
+        return null;
+      case '.kytos':
+        console.log("Kytos extension detected. Switch extension to .vue");
+        return null;
+      default: return undefined; // let vue3-sfc-loader handle this
+    }
+  },
+
+  async getFile(url) {
+
+    const res = await fetch(url);
+    if (!res.ok)
+      throw Object.assign(new Error(url + ' ' + res.statusText), { res });
```

Comment:
Did you have the chance to see this throw in practice when developing and exploring it? I'm wondering if it ever happens if it'll be obvious to spot or not